tests/test_util.py

```diff
@@ -45,3 +46,16 @@ def test_warn(capsys):
     util.warn('test warn')
     _, err = capsys.readouterr()
     assert '!!! WARNING: test warn' in err
+
+
+def test_import_app():
+    object_app = "support"
```

Comment:
Style nit: Rest of the file uses sing quotes. It would be nice to be consistent with the rest of the file.